enhancements/monitoring/replace-prometheus-adapter-with-metrics-server.md

```diff
@@ -0,0 +1,189 @@
+---
+title: replace-prometheus-adapter-with-metrics-server
+authors:
+  - "@slashpai"
+reviewers:
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+approvers: 
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-06-20
+last-updated: 2023-06-20
+tracking-link:
+  - https://issues.redhat.com/browse/MON-3153
+---
+
+# Replace prometheus-adapter with metrics-server
+
+## Summary
+
+This proposal suggests replacing the [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) currently used for monitoring resource metrics in OpenShift with the [metrics-server](https://github.com/kubernetes-sigs/metrics-server).
+
+## Motivation
+
+For autoscaling OpenShift needs a resource metrics implementation. Currently this depends on CMO's Prometheus stack.
+
+Reasons to consider to moving to metrics-server are:
+
+1. Initially we planned using [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) for custom metrics API too but the direction has now shifted towards KEDA [OCPNODE-708]((https://issues.redhat.com/browse/OCPNODE-708)) so there's little incentive from the OpenShift monitoring team's perspective to continue investing into prometheus-adapter.
+
+2. Deploying and configuring Prometheus Adapter requires additional effort compared to Metrics Server. It involves setting up and maintaining a Prometheus instance alongside the adapter. Some users would like to opt-out of running a fully fledged Prometheus stack, see [MON-3152](https://issues.redhat.com/browse/MON-3152).
+
+3. Utilizing Prometheus Adapter effectively may require some familiarity with Prometheus and its query language (PromQL). This learning curve could be a drawback for users not already familiar with Prometheus.
+
+3. [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) project will be archived soon due to difficulty in maintaining the project
```

Comment:
I think this should move to top and one of the main reasons to consider replacing prometheus-adapter with `metrics-server`